base/dates/conversions.jl

```diff
@@ -129,3 +117,15 @@ Takes the given `DateTime` and returns the number of Julian calendar days since
 epoch `-4713-11-24T12:00:00` as a `Float64`.
 """
 datetime2julian(dt::DateTime) = (value(dt) - JULIANEPOCH)/86400000.0
+
+function Base.convert(::Type{DateTime}, c::Libc.ComputerTime)
+    base = unix2datetime(c.seconds)
+    extra = Millisecond( round( c.microseconds / 1000 ) )
+    base + extra
+end
+
+function Base.string(c::Libc.ComputerTime)
```

Comment:
I'm not sure I understand. I tried to follow the strategy in base/dates/io.jl. I had intended to move these functions there; I can still do this.